CronResponce.php

```diff
@@ -46,9 +47,6 @@
     }
     
     $handle = fopen(_PS_MODULE_DIR_ . 'sendinblue/csv/SyncToSendinblue.csv', 'w');
-    $key_value = array();
-    $key_value[] = '';
-    fputcsv($handle, $key_value, '');
```

Comment:
This code is used to write blank data after processing as we don't want any once to access these data by using some URL.